bears/java/JavaPMDBear.py

```diff
@@ -78,21 +79,24 @@ def create_arguments(filename, file, config_file,
         :param allow_unused_code:
             Allows unused code.
         """
-        options = {
-            'java-basic': check_best_practices,
-            'java-braces': check_braces,
-            'java-clone': check_clone_implementation,
-            'java-codesize': check_code_size,
-            'java-comments': check_comments,
-            'java-controversial': check_controversial,
-            'java-design': check_design,
-            'java-imports': check_imports,
-            'java-naming': check_naming,
-            'java-optimizations': check_optimizations,
-            'java-strings': check_strings,
-            'java-unnecessary': not allow_unnecessary_code,
-            'java-unusedcode': not allow_unused_code}
-        rules = ','.join(key for key in options if options[key])
+        if pmd_config:
+            rules = pmd_config
```

Comment:
you probably have to load the file, don't you?
However, it's probably better to pass the filename directly to `pmd` instead of reading those rules :+1:

Reply:
I have done these changes in the latest commit.

Reply:
Still not addressed, see http://pmd.sourceforge.net/pmd-4.2.5/running.html on how to pass ruleset files.